detection:
  debug: false
  output: "color"
  minimum: 1000
  grid_width: 100
  grid_height: 100
  profiles:
    - name: green
      hue_up: 100
      sat_up: 255
      val_up: 255
      hue_low: 30
      sat_low: 30
      val_low: 30
